src/dashboard/src/requirements/base.txt

```diff
@@ -11,7 +11,7 @@ django-extensions==1.1.1
 django-autoslug==1.7.1
 django-annoying==0.7.7
 elasticsearch>=1.0.0,<2.0.0
-git+https://github.com/artefactual/archivematica-fpr-admin.git@v1.6.0#egg=archivematica-fpr-admin
+git+https://github.com/artefactual/archivematica-fpr-admin.git@020a713e54ff254d25d64432f8777bba2aaad63c#egg=archivematica-fpr-admin
```

Comment:
Yes, that makes sense. I merge upstream into qa/jisc periodically but I haven't done it since last week. This change will be included next time we do it.